src/main/java/frc/robot/subsystems/arm/commands/ArmJoystickControl.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class ArmJoystickControl extends CommandBase {
+    private final Arm arm;
+    private final Joystick shoulderJoystick;
+    private final Joystick elbowJoystick;
+
+    public ArmJoystickControl(Joystick shoulderJoystick, Joystick elbowJoystick) {
```

Comment:
It's better to pass suppliers of the throttle value instead of the joysticks themselves.